chpl/chpl-service/src/main/java/gov/healthit/chpl/web/controller/TestingLabController.java

```diff
@@ -159,20 +193,36 @@ public TestingLab create(@RequestBody TestingLab atlInfo) throws InvalidArgument
         address.setZipcode(atlInfo.getAddress().getZipcode());
         address.setCountry(atlInfo.getAddress().getCountry());
         toUpdate.setAddress(address);
-
+        
```

Comment:
There are some random whitespaces issues through these files that I've not been flagging. If you have a chance to clear them that's great, but not required. Checkstyle should have been flagging some of them

Reply:
I'll take a look at this.